core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -52,6 +52,9 @@ <h3>Review translation</h3>
 </div>
 
 <style>
+  .oppia-translation-modal-subheading {
+    margin: 17px 12px;
```

Comment:
Does the top and bottom need to be exactly 17px?  I'm wondering if you we can make this an even number.

Reply:
I guess we can make this 18px, but I chose this since it actually looks the best.

Reply:
I have made this 18px! It should make more sense.